pnet_datalink/src/lib.rs

```diff
@@ -23,30 +23,26 @@ pub use pnet_base::{MacAddr, ParseMacAddrErr};
 
 mod bindings;
 
-
 #[cfg(windows)]
 #[path = "winpcap.rs"]
 mod backend;
 
 #[cfg(windows)]
 pub mod winpcap;
 
-#[cfg(all(not(feature = "netmap"),
-          target_os = "linux"
-          )
-      )]
+#[cfg(all(not(feature = "netmap"), target_os = "linux"))]
```

Comment:
I actually prefered the formatting before the rustfmt here... `#[cfg]`'s are really hard to read.

Reply:
Ok, let me revert this one.